compiler/shared/test/diff/LambLift.mls

```diff
@@ -0,0 +1,105 @@
+:NewDefs
+
+:e // FIXME: Malformed Test?
```

Comment:
@yuankeyu could you comment on this and propose a fixa? What were you trying to do?

Reply:
For type errors, just change the `foo(x)` to `foo()` on line 7. This case can test my program of lifting nested functions and analyzing non-terminating programs, so it will yield RE. If RE should also be avoided then maybe just delete the `foo(x)` on line 7.